src/CGRA.cpp

```diff
@@ -170,8 +170,11 @@ CGRA::CGRA(int t_rows, int t_columns, bool t_diagonalVectorization,
     if (t_heterogeneity) {
```

Comment:
Enabling path merging shouldn't depend on whether we support `heterogeneity`, right?